src/components/spreadedComponents/StyledButton/StyledButton.tsx

```diff
@@ -0,0 +1,9 @@
+import { styledButtonProps } from "./interface/styledButton.interface"
+
+const StyledButton = ({ text }: styledButtonProps) => {
+    return (
+        <button className="styled-button" type="button">{text}</button>
```

Comment:
please pass handler to this component
in the places where you use it you may pass down empty function